File: Events/DataAccess/Models/Event.cs
namespace Events.Models;

public class Event
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string Plan { get; set; }
    
    public virtual Company Company { get; set; }
    public int CompanyId { get; set; }
    
    public virtual Speaker Speaker { get; set; }
    public int SpeakerId { get; set; }

    public DateTime Time { get; set; }
    public string Place { get; set; }
}

File: Events/ClientApp/src/app/_models/Event.ts
import {Time} from "@angular/common";

export class Event {
  constructor (
    public id: number,
    public title: string,
    public description: string,
    public plan: string,
    public date: string,
    public time: string,
    public place: string,
    public company: string,
    public speaker: string,
  ) {}
}



